zcash_client_backend/src/data_api/wallet/input_selection.rs

```diff
@@ -455,9 +455,9 @@ where
                 Err(other) => return Err(other.into()),
             }
 
-            #[cfg(not(zcash_unstable = "orchard"))]
+            #[cfg(not(feature = "orchard"))]
             let selectable_pools = &[ShieldedProtocol::Sapling];
-            #[cfg(zcash_unstable = "orchard")]
+            #[cfg(feature = "orchard")]
```

Comment:
Hmm: we now don't attempt to select from the Orchard pool if `orchard` is disabled. What was the previous behaviour that made this change necessary, given that the `wallet_db` method is publicly reachable and thus other people could also run into it?

Reply:
The objective here is to make the zcash_unstable only be a gate for the 'orchard' feature, which is then the ultimate gate (that will remain in place permanently) for Orchard functionality. Removing the other uses of the config flag means that we have less variation in compilation to account for.